tasks/download.sh

```diff
@@ -35,8 +35,19 @@ verify-file() {
 }
 
 download() {
+  local tmp_file_name="pe-tmp-file"
```

Comment:
Can probably omit this as a variable, not needed. The only place it's used is in the call to `mktemp`, where it's just a beautification argument anyway.